cryptol-saw-core/src/Verifier/SAW/Cryptol.hs

```diff
@@ -1913,7 +1914,7 @@ exportValue ty v = case ty of
         | TV.isTBit e -> V.VWord <$>
             V.bitmapWordVal V.Concrete (toInteger (Vector.length xs))
                  (V.finiteSeqMap V.Concrete . map (V.ready . SC.toBool . SC.runIdentity . force) $ Fold.toList xs)
-        | otherwise   -> pure . V.VSeq (toInteger (Vector.length xs)) $ V.finiteSeqMap V.Concrete $
+        | otherwise   -> V.mkSeq V.Concrete (C.Nat (toInteger (Vector.length xs))) e $ V.finiteSeqMap V.Concrete $
```

Comment:
This feels more complicated that it needs to be:

1. The `isTBit e` case above this one already checks that by the time we get to the `otherwise` case, the element type is not `Bit`.
2. In the `otherwise` case, we have to define a somewhat involved `ftToTValue` function, all for the sake of passing a `TValue` to `mkSeq`. The only reason that `mkSeq` needs a `TValue` is because `mkSeq` needs to check if the element type is `Bit` or not... but that was already checked above!

I think this would be an ideal use case for `mkFinSeq` (or `unsafeFinSeq`, or whatever we end up calling it) from https://github.com/GaloisInc/cryptol/pull/1804, which would avoid the need for any of this `ftToTValue` business.

Reply:
ftToTValue was only used once (mentioned below). This usage of mkSeq seems in line with the others.

Reply:
Ah, I attached this review comment to the wrong line. But in any case, I like the way this looks much better now that `ftToTValue` is gone.